src/core/gosling.schema.ts

```diff
@@ -254,16 +254,51 @@ interface RangeMouseEventData extends CommonEventData {
     genomicRange: [GenomicPosition, GenomicPosition] | null;
 }
 
+/** The visual parameters that determine the shape of a linear track */
+interface LinearTrackShape {
+    x: number; // The origin is the left top corner
+    y: number;
+    width: number;
+    height: number;
+}
```

Comment:
Maybe move this comment so that it is picked up by JSDoc.

```suggestion
/** 
 * The visual parameters that determine the shape of a linear track. 
 * Origin is the left top corner.
 */
interface LinearTrackShape {
    x: number;
    y: number;
    width: number;
    height: number;
}
```